docs/development/changing-the-api.md

```diff
@@ -33,7 +33,11 @@ It must be ensured that the API is always backwards-compatible.
 
 #### Removing a field
 
-If fields shall be removed permanently from the API then a proper deprecation period must be adhered to so that end-users have enough time adapt their clients. The followed process for removing a field is:
+If fields shall be removed permanently from the API then a proper deprecation period must be adhered to so that end-users have enough time to adapt their clients.
```

Comment:
```suggestion
If fields shall be removed permanently from the API, then a proper deprecation period must be adhered to so that end-users have enough time to adapt their clients.
```